git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -30,39 +39,39 @@ After completion of this module, learners will be able to:
 good_first_module: false
 collection: learn_to_code
 coding_required: true
+coding_level: basic
+coding_language: git, bash
+sequence_name: git_basics
+previous_sequential_module: git_setup_mac_and_linux, git_setup_windows
```

Comment:
@drelliche Not sure if having two previous sequential modules listed is acceptable or if it will throw a wrench into any of the discovery app stuff!

Reply:
verbal convo with Elizabeth right now confirmed that this is fine!